.github/workflows/base-component-descriptor.yaml

```diff
@@ -26,6 +26,23 @@ on:
         required: false
         type: string
         default: SAP SE
+      component-references:
+        required: false
+        type: string
+        description: |
+          relative path (to repo root) to YAML/Json file containing definition of
+          component-references added to base-component-descriptor, defaults to
+          `.ocm/component-references.yaml`.
+          action will only raise on absence of file if an explicit value is given.
+
+          Example contents:
+          - componentName: github.com/gardener/gardener
+            version: v1.110.0
+            name: gardener/gardener
+            # following key-values pairs are optional
+            digest: foo
```

Comment:
at least `foo` is not a valid digest - I would maybe rather drop it (also to make the example shorter), and rather put a link to OCM-Spec (about crefs) for further reading.